cmd/metricscollector/v1beta1/tfevent-metricscollector/main.py

```diff
@@ -38,6 +38,7 @@ def parse_options():
     logger.addHandler(handler)
     logger.propagate = False
     opt = parse_options()
+    wait_all_processes = opt.wait_all_processes.lower() == "true"
```

Comment:
Does this compare (`opt.wait_all_processes.lower() == "true"`) necessary ?
I assume `wait_all_processes` can be only "true" or "false"  since `WaitAllProcesses *bool` in Katib config?

Reply:
this is the python way of converting string to bool ;) should be fine since the string can only have two possibilities